packages/cells/src/widget.ts

```diff
@@ -1486,6 +1486,10 @@ export class MarkdownCell extends AttachmentsCell<IMarkdownCellModel> {
     });
     this.renderCollapseButtons(this._renderer!);
     this.renderInput(this._renderer!);
+    this._showEditorForReadOnlyMarkdown =
+      typeof options.showEditorForReadOnlyMarkdown === 'undefined'
+        ? MarkdownCell.defaultShowEditorForReadOnlyMarkdown
+        : options.showEditorForReadOnlyMarkdown;
```

Comment:
You could simplify this with the nullish operator:

```suggestion
      options.showEditorForReadOnlyMarkdown ?? MarkdownCell.defaultShowEditorForReadOnlyMarkdown;
```